roles/twig/templates/application.yml.j2

```diff
@@ -12,6 +12,7 @@ time-suffix: {{ twig.time_suffix | default('yyyyMMdd') }}
 session:
   default-limit: {{ twig.session.default_limit | default('50') }}
   use-x-correlation-header: {{ twig.session.use_x_correlation_header | default('true') }}
+  show-retransmits: {{ twig.session.show_retransmits | default('true') }}
```

Comment:
`sip3-twig-ce` doesn't use `session.call.show-retransmits` and requires only `session.show-retransmits`. That's why we may remove `session.call.show-retransmits` from template.